arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -2179,7 +2179,7 @@
 				3E0B58451E36BCCD00951767 /* Analysis */,
 				3EEE1D671E4935FC003263FC /* Cloud & Portal */,
 				3EEA06001D21C3E800E03774 /* Scenes */,
-				D98CC08F22E25D8200618682 /* Utility Network */,
+				D98CC08F22E25D8200618682 /* Utility network */,
```

Comment:
I am wondering if this change is necessary. You are updating this line but not updating Route & Directions and Cloud & Portal. Perhaps it is not necessary for the TOC listing?

Reply:
This change does not have effect on dev site TOC, but should be done as part of casing update.

In terms of `Route & Directions` and `Cloud & Portal` in project file, I didn't receive comments on whether we should make them consistent or keep them as-is, so I didn't change them. I tend to keep category names consistent everywhere, but I don't know who can decide on if we should make this move. Maybe wait for Phil's feedback? 🤔

Reply:
@maryharvey @vquach2404 I've addressed your feedback and make these category names unified everywhere. What do you think?

Reply:
I like that they're uniform but you might be right about the labels. Maybe we should consider keeping the '&' and only change the capitalization. It keeps the labels short and much more readable compared to having a long line of all letters for a label.